os/include/debug.h

```diff
@@ -724,6 +724,12 @@ int get_errno(void);
 #define pgllvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_SYSTEM_FAULTS
+#define fltdbg(format, ...) lldbg(format, ##__VA_ARGS__)
```

Comment:
Naming policy is different with others.
xxdbg is defined to dbg.
xxlldbg is defined to lldbg.
But fltdbg is defined to lldbg, not dbg.
How about using `fltlldbg`? Then we can easily know fltlldbg prints logs directly.

Reply:
Done.